test/e2e/utils/noderesourcetopology.go

```diff
@@ -259,3 +279,12 @@ func isValidResourceList(zoneName string, resources v1alpha2.ResourceInfoList) b
 	}
 	return foundCpu
 }
+
+func containsAttribute(attributes v1alpha2.AttributeList, attribute v1alpha2.AttributeInfo) bool {
+	for _, attr := range attributes {
+		if reflect.DeepEqual(attr, attribute) {
```

Comment:
minor: reflect is no bother, but can't we just use '=='?

Reply:
I was thinking about doing:
```go
if attr.Name == attribute.Name && attr.Value == attribute.Value {
    return true
}
```
but settle on shorter one.

Reply:
I would have used the explicit == version, but I don't have strong feelings so works for me.

Reply:
My thinking also that `attr == attribute` should've done the job but especially in the e2e tests I don't have that strong feelings either so let's go with this...